workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/MapQualitySettings.cs

```diff
@@ -0,0 +1,136 @@
+﻿﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Fps
+{
+    [CreateAssetMenu(fileName = "MapQualitySettings", menuName = "Improbable/Map Quality Settings")]
+    public class MapQualitySettings : ScriptableObject
+    {
+        public const float DefaultCheckoutDistance = 60f;
+
+        public static bool ShowPreview;
```

Comment:
Should this be static?

Reply:
Functionally, it doesn't matter...if you deselect the scriptable object is disables `ShowPreview` automatically. I would argue it's more indicative of its purpose as a static bool, which is that it doesn't care about any single instance of MapQualitySettings.